app/moves/controllers/list.test.js

```diff
@@ -87,7 +87,33 @@ describe('Moves controllers', function() {
         sinon.stub(permissions, 'check')
       })
 
-      context('if user can view move', function() {
+      context(
+        'if user can view move and individual location requested',
+        function() {
+          beforeEach(function() {
+            req.session = {
+              user: {
+                permissions: ['move:view'],
+              },
+            }
+            res.locals = {
+              fromLocationId: '83a4208b-21a5-4b1d-a576-5d9513e0b910',
+            }
+            permissions.check.withArgs('move:view', ['move:view']).returns(true)
+
+            controller(req, res)
+          })
+
+          it('should render list template', function() {
+            const template = res.render.args[0][0]
+
+            expect(res.render.calledOnce).to.be.true
```

Comment:
This can also be written as `expect(res.render).to.be.calledOnce`. Sometimes I find it's a little easier to read using this style.